### ISSUE TITLE
[BUG] Help output is printed slowly

### ISSUE THREAD

**Issue by onox** [pyjarrett/septum#39]
**Describe the bug**
The output printed by `help` is printed quite slowly.

**To Reproduce**
1. Go to the prompt
2. Type 'help' and press the enter key
3. It takes like 1 or 2 seconds to print everything

**Expected behavior (if applicable)**
See the help text getting printed more or less instantly

**Desktop (please complete the following information):**
 - OS: Arch Linux
 - Version: Built from Source

**Additional context**
Minor issue.

**Comment by pyjarrett** [comment]
Might be a regression of #35, I think it might be buffering due to not using a back buffer and the current Trendy Terminal newline settings instead of scrolling.  Last time IIRC it was related to a long delay due to `VT100.Report_Cursor_Position` procedure.

**Comment by onox** [comment]
Using https://github.com/onox/goblin-terminal (it uses VTE like gnome-terminal). But output is printed instant in tmux! :open_mouth:

**Comment by pyjarrett** [comment]
Yup, if you use `strace` to check the length of the system calls you might see an issue.  `htop` has an easy way of visualizing this.

**Comment by onox** [comment]
For each line printed, I see:

- read()
- read() returning E_AGAIN
- write()
- 4x poll()
- write()
- clock_nanosleep()
- 2x poll()
- write()

With weston-terminal the output is printed instantly and I see:

- epoll_wait()
- read()
- write()

**Comment by onox** [comment]
Slow in gnome-terminal as well (not a surprise since it uses VTE)

**Comment by pyjarrett** [comment]
Thanks!  I'll look when I can, but it might be a couple of days before I get to this.

**Comment by pyjarrett** [comment]
There's been a lot of changes and I've fixed some things in this area including one inefficiency, but I was never able to reproduce this.

**Comment by pyjarrett** [comment]
Reproduced error with Terminator, fixed in 2eef8a3a50be5f8eecc848f1b204f930f687a18d
